pkg/cluster/k8sres.go

```diff
@@ -769,7 +769,13 @@ func (c *Cluster) generateSpiloPodEnvVars(
 	cloneDescription *acidv1.CloneDescription,
 	standbyDescription *acidv1.StandbyDescription,
 	customPodEnvVarsList []v1.EnvVar) []v1.EnvVar {
-	envVars := []v1.EnvVar{
+	envVars := make([]v1.EnvVar, 0)
+
+	if len(c.Spec.Env) > 0 {
+		envVars = append(envVars, c.Spec.Env...)
+	}
```

Comment:
Appending c.Spec.Env should happen after the hard-coded set of variables, because we don't want them to get overridden by custom variables.

Reply:
Ok, good point, thanks. WIll implement